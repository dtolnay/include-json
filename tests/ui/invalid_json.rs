use include_json::include_json;

fn main() {
    let _ = include_json!(concat!(env!("CARGO_MANIFEST_DIR"), "/../../../../tests/ui/invalid_json.json"));
}
