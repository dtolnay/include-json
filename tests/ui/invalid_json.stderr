error: lone leading surrogate in hex escape at line 2 column 9
 --> tests/ui/invalid_json.rs:4:13
  |
4 |     let _ = include_json!(concat!(env!("CARGO_MANIFEST_DIR"), "/../../../../tests/ui/invalid_json.json"));
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_json` (in Nightly builds, run with -Z macro-backtrace for more info)
